tools/downloader/src/open_model_zoo/model_tools/_configuration.py

```diff
@@ -402,8 +424,12 @@ def load_models_or_die(args):
 # requires the --print_all, --all, --name and --list arguments to be in `args`
 def load_models_from_args(parser, args):
     if args.print_all:
+        model_names = set()
         for model in load_models_or_die(args):
-            print(model.name)
+            name_to_print = model.name if model.composite_model_name is None else model.composite_model_name
```

Comment:
Composite model names should not be printed here. This breaks the invariant that for every name printed, there is a corresponding entry in the info dumper output.

(I think potentially it could be a good idea for info dumper to have an option to print information about composite models, but this should be done in a backwards-compatible manner.)